src/components/MultiStepIndicator/components/StepIndicator/StepIndicator.tsx

```diff
@@ -86,6 +116,7 @@ const StepIndicator = ({
 
   // Event listeners for removing animation.
   useEventListener({
+    // @ts-ignore TODO either fix the import to 'useEventListener' OR fix to fit 'useKeyEvent' OR remove entirely
```

Comment:
needs to be discussed
it seems that as for today, since it uses wrong import (`useEventListener` is imported mistakenly from `useKeyListener`), the api is wrong.
`useKeyListener` does not have `eventName`, but rather `keyEventName`, causing `animationend` event to not be assigned to `componentRef`, and instead assigning it with `keydown` event.

This does not seem to have effect eventually as `disableStatusChangeAnimation` is never called (but weirdly, the component never gets the `withAnimation` class declared above 🥲 plz help 🥲)

Reply:
Sound good to me.  Can you open a task for it in our backlog with link to here?

Reply:
https://monday.monday.com/boards/3532714909/pulses/4817660390